wled00/bus_manager.h

```diff
@@ -17,11 +17,12 @@ struct BusConfig {
   uint16_t start = 0;
   uint8_t colorOrder = COL_ORDER_GRB;
   bool reversed = false;
+  uint16_t clkspeed;
   uint8_t pins[5] = {LEDPIN, 255, 255, 255, 255};
-  BusConfig(uint8_t busType, uint8_t* ppins, uint16_t pstart, uint16_t len = 1, uint8_t pcolorOrder = COL_ORDER_GRB, bool rev = false) {
-    type = busType; count = len; start = pstart; colorOrder = pcolorOrder; reversed = rev;
+  BusConfig(uint8_t busType, uint8_t* ppins, uint16_t pstart, uint16_t len = 1, uint8_t pcolorOrder = COL_ORDER_GRB, bool rev = false, uint16_t clkSpeed = 5001UL) {
```

Comment:
Why is it `5001UL` instead of `5000UL` here?

Reply:
I used unique values for debugging, then missed this one during cleanup